src/widgets/modals/PageContentModal.js

```diff
@@ -38,41 +27,28 @@ export class PageContentModal extends React.Component {
   }
 
   render() {
-    const {
-      onClose,
-      title,
-      style,
-      ...modalProps,
-    } = this.props;
-
-    const titleComponent = title && (
-      <View style={localStyles.titleContainer}>
-        <Text style={localStyles.title}>{title}</Text>
-      </View>
-    );
+    const { onClose, title, style, ...modalProps } = this.props;
 
     return (
-      <BottomModal
-        {...modalProps}
-        style={[localStyles.modal, style]}
-      >
-        {titleComponent}
-        <TouchableOpacity onPress={onClose} style={localStyles.closeButton}>
-          <Icon name="md-close" style={localStyles.closeIcon} />
-        </TouchableOpacity>
-        <View style={localStyles.childrenContainer}>
-          {this.props.children}
+      <BottomModal {...modalProps} style={[localStyles.modal, style]}>
+        <View style={localStyles.titleContainer}>
+          <View style={{ flex: 1 }} />
```

Comment:
I really like how const titleComponent was used, can we refactor this one please